### PR TITLE
[missiontimer] fix for "server triggered client side event XYZ but event is not added clientside"

### PR DESCRIPTION
**Pull request by Einheit-101**

### CODE REVIEW

File: missiontimer_server.lua [hunk 1]
```diff
@@ -0,0 +1,147 @@
+﻿local thisResource = getThisResource()
+local bool = { [false]=true, [true]=true }
```
Comment (Necktrox) [inline missiontimer_server.lua]: You can use `type(var) == "boolean"` instead

File: missiontimer_server.lua [hunk 2]
```diff
@@ -0,0 +1,147 @@
+﻿local thisResource = getThisResource()
```
Comment (Necktrox) [inline missiontimer_server.lua]: MTA offers a predefined variable called `resourceRoot`, which you could've used

File: missiontimer_server.lua [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+addEventHandler("onPlayerQuit",root,
+	function()
+		table.removevalue( readyPlayerList, source )
+	end
+)
+
+addEventHandler ("onResourceStop",root,
+	function()
+		for i,timer in ipairs(getElementsByType("missiontimer",source)) do
```
Comment (Necktrox) [inline missiontimer_server.lua]: Replace `ipairs` with `pairs`, because it's faster and the order does not matter

**Comment by Einheit-101** [comment]
Yup, all that stuff doesnt belong to me, its from the original author of missiontimer. I implemented your modifications.

**Comment by qaisjp** [comment]
Thanks for the PR. the content hasn't been checked because you have not edited the existing resource, you have to uploaded the file to the root of your repository.

See: https://github.com/Einheit-101/mtasa-resources/tree/(missiontimer)

Please submit a new pull request with this fix. Thanks!

**Pull request closed without merging**